joebvp/makevoigt.py

```diff
@@ -85,9 +85,13 @@ def get_lsfs():
 
     lsfobjs=[]
     for i,inst in enumerate(cfg.instr):
-        lsfobjs.append(LSF(dict(name=inst, grating=cfg.gratings[i],
+        if inst in ['COS','STIS']:
+            lsfobjs.append(LSF(dict(name=inst, grating=cfg.gratings[i],
                                 life_position=cfg.lps[i], cen_wave=cfg.cen_wave[i],
```

Comment:
does STIS have different `life_position`?

Reply:
STIS does not

Reply:
I know it will be ignored, but it would be cleaner if you separate the cases: one for COS, and another for STIS (similarly to what is done between these two and a Gaussian LSF)...